extensions/sql-migration/src/wizard/wizardController.ts

```diff
@@ -114,11 +117,14 @@ export function createHeadingTextComponent(view: azdata.ModelView, value: string
 }
 
 
-export function creaetLabelTextComponent(view: azdata.ModelView, value: string): azdata.TextComponent {
+export function createLabelTextComponent(view: azdata.ModelView, value: string, styles?: { [key: string]: string; }): azdata.TextComponent {
+	if (!styles) {
```

Comment:
You can just specify a default in the parameter list 

`styles: { [key: string]: string; } = { 'width': '300px' }`